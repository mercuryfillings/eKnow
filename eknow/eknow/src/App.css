@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;700&display=swap');

body {
  background-image: url("Airports.jpg");
  background-position: center;
}

.card {
  width: auto;
  height: 60%;
  background-color: rgba(228, 228, 228, 0.7);
  left: 15px;
  right: 15px;
  margin: 15px;
  padding: 15px;
  top: 15px;
  bottom: 15px;
  border-radius: 8px;
  box-shadow: 0px 10px 30px 3px rgba(0, 0, 0, 0.55);
}

h1 {
  margin: 0;
  padding: 20px;
  color: white;
}

h2 {
  text-align: center;
  font-family: Quicksand;
  font-weight: 300;
  font-size: 35px;
  color: rgb(65, 65, 65)
}

.enopic {
  width: 300px;
  height: auto;
  margin-left: -55px;
}

.logocontainer {
  overflow: hidden;
  height: 200px;
  width: 200px;
  border-radius: 50%;
  box-shadow: 0px 3px 8px 3px rgba(0, 0, 0, 0.55);
  margin: 0 auto;
}