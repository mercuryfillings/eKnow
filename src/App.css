@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;700&display=swap');

body {
  background-image: url("Airports.jpg");
  background-position: center;
  background-attachment: fixed;
}

.fade-enter {
  opacity: 0;
  z-index: 1;
}

.fade-enter.fade-enter-active {
  opacity: 1;
  transition: opacity 350ms ease-in;
}

.card {
  width: auto;
  height: 60%;
  background-color: rgba(228, 228, 228, 0.7);
  left: 15px;
  right: 15px;
  margin: 15px;
  padding: 15px;
  top: 15px;
  bottom: 15px;
  border-radius: 8px;
  box-shadow: 0px 10px 30px 3px rgba(0, 0, 0, 0.55);
}

h1 {
  margin: 0;
  padding: 20px;
  color: white;
}

h2 {
  text-align: center;
  font-family: Quicksand;
  font-weight: 300;
  font-size: 35px;
  color: rgb(65, 65, 65)
}

.enopic {
  width: 300px;
  height: auto;
  margin-left: -55px;
}

.logocontainer {
  overflow: hidden;
  height: 200px;
  width: 200px;
  border-radius: 50%;
  box-shadow: 0px 3px 8px 3px rgba(0, 0, 0, 0.55);
  margin: 20px auto 20px auto;
}

.footer {
  text-align: center;
  font-family: Quicksand;
  font-weight: 300;
  font-size: 12px;
  background-color: rgba(255,255,255,.7);
  border-radius: 10px 10px 0 0;
  color: rgb(65, 65, 65);
  width: 45%;
  margin: 0 auto;
  padding: 5px;
}

.foot {
  margin: 10px;
}

.copy {
  text-align: center;
  font-family: Quicksand;
  font-weight: 300;
  font-size: 15px;
  color: rgb(65, 65, 65);
  margin: 0 auto;
  width: 70%;
}