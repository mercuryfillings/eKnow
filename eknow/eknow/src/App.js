import React, { useState } from 'react';
import { Link, Route, Switch } from 'react-router-dom'
import './App.css';
import ProblemInput from './ProblemInput'
import ProblemPage from './ProblemPage'
import Header from './Header'

function App() {

  const [problem, setProblem] = useState('');

  function handleChange(newProblem) {
    setProblem(newProblem);
  }

  return (
    <div className="App">
      <header className="App-header">
        <Header />
      </header>
      <main>
        <Switch>
          <Route exact path='/'>
            <section className="card">
              <h2>ASK ENO WHAT HE KNOW</h2>
              <div className='logocontainer'>
                <img className='enopic' src={require('./Eno.jpg')} alt='Brian Eno' title='Brian Eno' /></div>
              <ProblemInput problem={problem} onChange={handleChange} />
              {/* <ul>
              <li>{problem}</li>
            </ul> */}
            </section>
          </Route>
          <Route path="/problem/" exact>
            <ProblemPage problem={problem} onChange={handleChange} />
          </Route>
        </Switch>
      </main>
    </div>
  );
}

export default App;
