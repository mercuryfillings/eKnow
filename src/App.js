import React, { useState, useEffect } from 'react';
import { Route, Switch } from 'react-router-dom'
import './App.css';
import ProblemInput from './ProblemInput'
import ProblemPage from './ProblemPage'
import ProblemList from './ProblemList'
import Footer from './Footer'
import Header from './Header'
import { TransitionGroup, CSSTransition } from 'react-transition-group'

function App() {

  const [problem, setProblem] = useState('');
  const [list, setList] = useState([]);
  const [musings, setMusings] = useState([]);

  function handleChange(newProblem) {
    setProblem(newProblem);
  }

  return (
    <div>
      <Route render={({ location }) => (
        <div>
          <header>
            <Header />
          </header>
          <main>
            <TransitionGroup>
              <CSSTransition
                key={location.key}
                timeout={300}
                classNames='fade'
              >
                <Switch>
                  <Route exact path='/'>
                    <section className="card">
                      <h2>ASK ENO WHAT HE KNOW</h2>
                      <div className='logocontainer'>
                        <img className='enopic' src={require('./Eno.jpg')} alt='Brian Eno' title='Brian Eno' /></div>
                      <ProblemInput problem={problem} onChange={handleChange} />
                      <ProblemList problem={problem} list={list} setList={setList} />
                      <p className="copy">Oblique Strategies is a card-based method for promoting creativity created by Brian Eno. Each card offers a challenging constraint intended to help artists (particularly musicians) break creative blocks by encouraging lateral thinking. This app uses these cards as a frame for a one-way conversation with yourself. </p>
                    </section>
                  </Route>
                  <Route path={'/' + problem} exact>
                    <ProblemPage problem={problem} onChange={handleChange} musings={musings} setMusings={setMusings} />
                  </Route>
                </Switch>
              </CSSTransition>
            </TransitionGroup>
          </main>
        </div>
      )} />
      <section className="foot">
        <Footer />
      </section>
    </div>
  )
}

export default App;
