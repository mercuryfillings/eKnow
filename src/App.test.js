import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const component = renderer.create(<App />).toJSON()
  expect(component).toMatchSnapshot()
})