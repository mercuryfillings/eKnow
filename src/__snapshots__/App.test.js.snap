// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<div>
  <p
    className="footer"
  >
    Powered by 
    <a
      href="http://brianeno.needsyourhelp.org/"
      target="_blank"
    >
      The Oblique Strategies API
    </a>
     and 
    <a
      href="https://picsum.photos/?ref=public-apis"
      target="_blank"
    >
      Lorem Picsum
    </a>
  </p>
</div>
`;

exports[`Header render snapshot 1`] = `
<div>
  <nav
    className="navbar"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <h1
        className="logo"
      >
        eKnow
      </h1>
    </a>
  </nav>
</div>
`;

exports[`Homepage renders Homepage render snapshot 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Homepage renders Problem page render snapshot 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Homepage renders Problem page snapshot 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Homepage renders Problem page snapshot correctly 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Homepage renders renders correctly 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Homepage renders renders correctly 2`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Page render checks Footer render snapshot 1`] = `
<div>
  <nav
    className="navbar"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <h1
        className="logo"
      >
        eKnow
      </h1>
    </a>
  </nav>
</div>
`;

exports[`Page render checks Footer snapshot 1`] = `
<div>
  <p
    className="footer"
  >
    Powered by 
    <a
      href="http://brianeno.needsyourhelp.org/"
      target="_blank"
    >
      The Oblique Strategies API
    </a>
     and 
    <a
      href="https://picsum.photos/?ref=public-apis"
      target="_blank"
    >
      Lorem Picsum
    </a>
  </p>
</div>
`;

exports[`Page render checks Header render snapshot 1`] = `
<div>
  <nav
    className="navbar"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <h1
        className="logo"
      >
        eKnow
      </h1>
    </a>
  </nav>
</div>
`;

exports[`Page render checks Header render snapshot 2`] = `
<div>
  <nav
    className="navbar"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <h1
        className="logo"
      >
        eKnow
      </h1>
    </a>
  </nav>
</div>
`;

exports[`Page render checks Homepage render snapshot 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Page render checks Homepage render snapshot 2`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Page render checks Input render snapshot 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Page render checks List render snapshot 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;

exports[`Page render checks should render 0 1`] = `
<div>
  <p
    className="footer"
  >
    Powered by 
    <a
      href="http://brianeno.needsyourhelp.org/"
      target="_blank"
    >
      The Oblique Strategies API
    </a>
     and 
    <a
      href="https://picsum.photos/?ref=public-apis"
      target="_blank"
    >
      Lorem Picsum
    </a>
  </p>
</div>
`;

exports[`Page renders Homepage render snapshot 1`] = `
<div>
  <div>
    <header>
      <div>
        <nav
          className="navbar"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo"
            >
              eKnow
            </h1>
          </a>
        </nav>
      </div>
    </header>
    <main>
      <div>
        <section
          className="card"
        >
          <h2>
            ASK ENO WHAT HE KNOW
          </h2>
          <div
            className="logocontainer"
          >
            <img
              alt="Brian Eno"
              className="enopic"
              src="Eno.jpg"
              title="Brian Eno"
            />
          </div>
          <form
            className="input"
            onSubmit={[Function]}
          >
            <input
              className="field"
              maxLength="255"
              onChange={[Function]}
              onSubmit={[Function]}
              placeholder="Enter a problem."
              type="text"
            />
            <input
              className="button"
              type="submit"
              value="START"
            />
          </form>
          <div>
            <ul>
              <li
                className="emptyprob"
              >
                <a
                  href="/"
                  onClick={[Function]}
                >
                  
                </a>
              </li>
            </ul>
          </div>
        </section>
      </div>
    </main>
  </div>
  <section
    className="foot"
  >
    <div>
      <p
        className="footer"
      >
        Powered by 
        <a
          href="http://brianeno.needsyourhelp.org/"
          target="_blank"
        >
          The Oblique Strategies API
        </a>
         and 
        <a
          href="https://picsum.photos/?ref=public-apis"
          target="_blank"
        >
          Lorem Picsum
        </a>
      </p>
    </div>
  </section>
</div>
`;
